src/components/Popup.js

```diff
@@ -1,9 +1,10 @@
 export default class Popup {
-  constructor(popup) {
+  constructor(popup, submitButton) {
     this._popupElement = document.querySelector(popup);
     this._popupCloseButton = this._popupElement.querySelector(
       ".modal__close-button"
     );
+    this._submitButton = document.querySelector(submitButton);
```

Comment:
`    this._submitButton = document.querySelector(submitButton);
`

Please, delete it from `Popup` because the image modal doesn't have a submit button